src/System.Data.SqlClient/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,7 @@
+MembersMustExist : Member 'System.Data.SqlClient.SqlBulkCopy.WriteToServer(System.Data.Common.DbDataReader)' does not exist in the implementation but it does exist in the contract.
```

Comment:
The project.json was referencing the net45 targetingpack even for net46(https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/project.json)
I made the change to include net46 targetingpack in case of net46 and the baseline file isn't needed.
the net45 baseline txt is needed however.